plugins/catalog-backend-module-msgraph/src/microsoftGraph/client.ts

```diff
@@ -167,6 +172,11 @@ export class MicrosoftGraphClient {
     return await fetch(url, {
       headers: {
         Authorization: `Bearer ${token.accessToken}`,
+
+        // Eventual consistency is required to use $search.
+        // Groups/Users are not changed that frequently to require strong consistency
+        // If a new user/group is not found, it'll eventually be imported on a subsequent call
+        ConsistencyLevel: 'eventual',
```

Comment:
would it be possible to set this _only_ if search is actually used?

Reply:
Yes. Done.